garden/src/features/gardens/api/usePatchGarden.ts

```diff
@@ -56,11 +56,13 @@ export const usePatchGarden = () => {
       return { previousGarden };
     },
     onError: (err, { doi }, context) => {
-      // If the mutation fails, use the context to roll back
-      if (context?.previousGarden) {
-        queryClient.setQueryData(["garden", doi], context.previousGarden);
+      console.error(err.message)
+      if (err.message.includes("409")) {
+        toast.error(`Failed to update garden: Garden must have at least one Model Author or one Gardener`);
```

Comment:
is there an easy way to have this just be the `detail` part of the response? since we're returning the same message from the backend anyway